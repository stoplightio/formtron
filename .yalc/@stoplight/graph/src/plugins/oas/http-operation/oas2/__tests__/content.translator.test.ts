import { translateToContent } from '../helpers/content.translator';

describe('content.translator', () => {
  test('should translate a simple content', () => {
    expect(
      translateToContent(
        {
          any: 'key',
          works: 'fine',
        },
        ['application/json', 'text/plain']
      )
    ).toMatchSnapshot();
  });
});
