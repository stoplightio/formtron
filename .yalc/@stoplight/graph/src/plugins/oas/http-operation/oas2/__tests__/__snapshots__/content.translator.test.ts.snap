// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`content.translator should translate a simple content 1`] = `
Object {
  "application/json": Object {
    "mediaType": "application/json",
    "schema": Object {
      "any": "key",
      "works": "fine",
    },
  },
  "text/plain": Object {
    "mediaType": "text/plain",
    "schema": Object {
      "any": "key",
      "works": "fine",
    },
  },
}
`;
